aten/src/ATen/native/native_functions.yaml

```diff
@@ -1733,6 +1733,9 @@
   variants: method
   device_guard: false
 
+- func: one_hot(IndexTensor self, int64_t num_classes=0) -> Tensor
```

Comment:
Does `int64_t?` work? It's a little weird to pass in num_classes=0 from the C++ API